app/src/main/res/values/string-not_translatable.xml

```diff
@@ -241,6 +243,91 @@ work. If not, see <https://creativecommons.org/publicdomain/zero/1.0/>.
+    <string name="abid_asciidoc_italic" translatable="false">abid_asciidoc_italic</string>
+    <string name="abid_asciidoc_monospace" translatable="false">abid_asciidoc_monospace</string>
+
+    <string name="abid_asciidoc_underline" translatable="false">abid_asciidoc_underline</string>
+    <string name="abid_asciidoc_overline" translatable="false">abid_asciidoc_overline</string>
+    <string name="abid_asciidoc_linethrough" translatable="false">abid_asciidoc_linethrough</string>
+    <string name="abid_asciidoc_nobreak" translatable="false">abid_asciidoc_nobreak</string>
+    <string name="abid_asciidoc_nowrap" translatable="false">abid_asciidoc_nowrap</string>
+    <string name="abid_asciidoc_prewrap" translatable="false">abid_asciidoc_prewrap</string>
+
+    <string name="abid_asciidoc_highlight" translatable="false">abid_asciidoc_highlight</string>
+
+    <string name="abid_asciidoc_subscript" translatable="false">abid_asciidoc_subscript</string>
+    <string name="abid_asciidoc_superscript" translatable="false">abid_asciidoc_superscript</string>
+
+    <string name="abid_asciidoc_break_thematic" translatable="false">abid_asciidoc_break_thematic</string>
+    <string name="abid_asciidoc_break_page" translatable="false">abid_asciidoc_break_page</string>
+
+    <string name="abid_asciidoc_squarebrackets" translatable="false">abid_asciidoc_squarebrackets</string>
+
+    <string name="abid_asciidoc_block_comment" translatable="false">abid_asciidoc_block_comment</string>
+    <string name="abid_asciidoc_block_example" translatable="false">abid_asciidoc_block_example</string>
+    <string name="abid_asciidoc_block_listing" translatable="false">abid_asciidoc_block_listing</string>
+    <string name="abid_asciidoc_block_literal" translatable="false">abid_asciidoc_block_literal</string>
+    <string name="abid_asciidoc_block_open" translatable="false">abid_asciidoc_block_open</string>
+    <string name="abid_asciidoc_block_sidebar" translatable="false">abid_asciidoc_block_sidebar</string>
+    <string name="abid_asciidoc_block_table" translatable="false">abid_asciidoc_block_table</string>
+    <string name="abid_asciidoc_block_pass" translatable="false">abid_asciidoc_block_pass</string>
+    <string name="abid_asciidoc_block_quote" translatable="false">abid_asciidoc_block_quote</string>
+
+    <string name="abid_asciidoc_insert_image" translatable="false">abid_asciidoc_insert_image</string>
+    <string name="abid_asciidoc_insert_link" translatable="false">abid_asciidoc_insert_link</string>
+    <string name="abid_asciidoc_table_insert_columns" translatable="false">abid_asciidoc_table_insert_columns</string>
+
+    <string name="abid_asciidoc_special_key" translatable="false">abid_asciidoc_special_key</string>
+
+    <!-- used in runAsciidocSpecialKeyAction -->
+
+    <!-- https://docs.asciidoctor.org/asciidoc/latest/text/text-span-built-in-roles/ -->
+    <string name="asciidoc_underline"      translatable="false">AsciiDoc role underline</string>
```

Comment:
Don't know where you plan to show that, but if you are in AsciiDoc format, users don't need to see `AsciiDoc whatever - prefix Underline`, but just `Underline`. And for example the string underline is already translated as R.string.underline.

Reply:
As I explained - there are block formattings and inline role formattings. And because they are listed in the extra menu, I want them to contain  "block" or  "role".
But maybe this will e adapted, because I have seen, in Markor I could also call sub menus like when inserting files. But I can't implement all at one time. So for now these are menu entries in the AsciiDoc special menu:
![image](https://user-images.githubusercontent.com/9020187/194130100-b6b712e7-a2a5-4957-a556-cb25483ee4fd.png)

![image](https://user-images.githubusercontent.com/9020187/194130467-0cff73cc-c1cd-48ae-bbbf-4486f0e34496.png)

it is currently a draft. Until I find better ways.

Reply:
it's fine to exactly do this way you do and want to. You make up and define the user experience for AsciiDoc, and you probably will be the "majority of overall use" of it for while with what you make here.

But you can use just words without AsciiDoc in string prefix, and use for example just generic "Underline" for "AsciiDoc Underline". And thus use already existing, translated-into-many-languages strings.

Just like when you have a Markdown file, there is the Bold button, named just Bold, not "Markdown Bold".

Reply:
Of course! You are right! They have been a kind of placeholders. I will change this a bit. I see my contribution as "alpha" AsciiDoc and my main goal was to implement basic features. Thank you for qualified and constructive feedback!

Reply:
Welcome. Well  thats what merge reviews are for, to improve it 😄 .